src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+
+```
+curl -X 'PATCH' -H 'Content-Type: application/json' -H "Authorization: Bearer $<var>YOUR_TOKEN</var>" 'https://scim-provisioning.service.newrelic.com/scim/v2/Groups/df2b9a04-0426-4a3e-bf5f-54d5341f4e5b' --data-binary @- <<EOF
+{
+  "schemas": [
+    "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+  ],
+  "Operations": [{
+    "op": "Add",
+    "path": "members",
+    "value": [{
+      "value": "5a1d580f-323c-450c-8c62-479b5c9085d6"
+    }]
+  }]
+}
+EOF
+```
+
+Example response:
+
+<Collapser title="200 OK">
+```
+{
+  "schemas": [
+    "urn:ietf:params:scim:schemas:core:2.0:Group"
+  ],
+  "id": "df2b9a04-0426-4a3e-bf5f-54d5341f4e5b",
+  "displayName": "Example Group",
+  "meta": {
+    "resourceType": "Group",
+    "created": "2021-07-20T22:41:50.414Z",
+    "lastModified": "2021-07-20T22:41:50.414Z"
+  },
+  "members": [
+    {
+      "type": "User",
+      "value": "5a1d580f-323c-450c-8c62-479b5c9085d6"
+    }
+  ]
+}
+```
+</Collapser>
+
+And to remove a user from a group, use the following operation parameters:
+* `op` set to `Remove`
+* `path` set to `members`
+* `value` set to a list of `{"value": "${USER_ID}"}` with each user ID to remove from the group
+
+```
+curl -X 'PATCH' -H 'Accept: application/json' -H "Authorization: Bearer $<var>YOUR_TOKEN</var>" 'https://scim-provisioning.service.newrelic.com/scim/v2/Groups/df2b9a04-0426-4a3e-bf5f-54d5341f4e5b' --data-binary @- <<EOF
+{
+  "schemas": [
+    "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+  ],
+  "Operations": [{
+    "op": "Remove",
+    "path": "members",
+    "value": [{
+      "value": "5a1d580f-323c-450c-8c62-479b5c9085d6"
+    }]
+  }]
+}
+EOF
+```
+
+Example response:
+
+<Collapser title="200 OK">
+```
+{
+  "schemas": [
+    "urn:ietf:params:scim:schemas:core:2.0:Group"
+  ],
+  "id": "df2b9a04-0426-4a3e-bf5f-54d5341f4e5b",
+  "displayName": "Example Group",
+  "meta": {
+    "resourceType": "Group",
+    "created": "2021-07-20T22:41:50.414Z",
+    "lastModified": "2021-07-20T22:41:50.414Z"
+  },
+  "members": []
+}
+```
+</Collapser>
+
+## Delete users and groups [#delete]
+
+To remove a user from an authentication domain, send a `DELETE` request to `/scim/v2/Users/${ID}`.
+
+```
+curl -X 'DELETE' -H 'Accept: application/json' -H "Authorization: Bearer $<var>YOUR_TOKEN</var>" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users/d0f4d8e3-5413-4894-a8f9-de709994e18c'
+```
+
+Example response:
+
+<Collapser title="204 No Content">
+</Collapser>
+
+Similarly, to remove a group from your authentication domain, send a `DELETE` request to `/scim/v2/Groups/${ID}`.
+
+```
+curl -X 'DELETE' -H 'Accept: application/json' -H "Authorization: Bearer $<var>YOUR_TOKEN</var>" 'https://scim-provisioning.service.newrelic.com/scim/v2/Groups/df2b9a04-0426-4a3e-bf5f-54d5341f4e5b'
+```
+
+Example response:
+
+<Collapser title="204 No Content">
+</Collapser>
+
+## Next steps [#next-steps]
+
+Once your integration is complete, next steps include: 
```

Comment:
I prefer something more active than "next steps include". How about

"Once you've completed your integration, review these next steps"